modules/op-about-triggers.adoc

```diff
@@ -0,0 +1,22 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/op-creating-applications-with-cicd-pipelines.adoc
+
+
+[id="about-triggers_{context}"]
+= About Triggers
+
+_Pipeline Triggers_ enables you to capture the external events and process them to extract key pieces of information. The extracted information from the received event payload is mapped to a set of predefined parameters, which trigger a series of tasks that may involve creation and deployment of the Kubernetes resources. By using Triggers in conjunction with the Pipelines, you can easily create a full-fledged CI/CD systems where the execution is defined entirely through Kubernetes resources.
+
```

Comment:
Suggest restructuring the paragraph: First sentence: You can use Triggers in conjunction with the Pipelines to easily create a fully-fledged CI/CD systems where the execution is defined entirely through Kubernetes resources.
Second sentence as is.
Third sentence: _Pipeline Triggers_ capture the external events

Reply:
Making the first sentence as the last sentence was breaking the continuity. Have reworded to make sense.